.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "mocha": true
+  },
+  "extends": "airbnb",
+  "rules": {
```

Comment:
We probably should alphabetize these to keep consistency and make it easy to find rules.